### DISCUSSION TITLE
See all Träwel Routes on a Singular Map

### DISCUSSION THREAD

**Discussion by ErikUden** [Traewelling/traewelling#2050] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
No problem/bug - just want all Träwels which can individually be seen on a map to be displayed as a whole on one singular map!

**Describe the solution you'd like**
*An example of a singular Träwel*
![Screenshot_20230610_120800_Trwelldroid.jpg](https://github.com/Traewelling/traewelling/assets/52011431/4ec3c289-0566-4f3c-8860-c013f08938a3)

Träwelling already displays every singular Träwel taken on a singular map! I think this is a very neat feature. However... 

**How I'd like it to look like:**
![Screenshot_20230610_121238_WebApps.jpg](https://github.com/Traewelling/traewelling/assets/52011431/c44dfd33-8a1e-4dca-967b-ff4361bf6f3b)

In [Travelynx](https://github.com/derf/travelynx) this feature already exists! One singular Leaflet OpenStreetMap showing all checked in train routes ever taken! It'd be awesome if Träwelling could do the same so one can look at all the Träwels they've taken overall, in a year, in a month, etc. 

Maybe there should also be a feature to filter and only show regional, local, ICEs etc. trains, filter by method of transportation (Train, Tram, Bus), or whether it was a personal / business Träwel!

To me, such filter options are great, but my main squeeze is that grand map showing all Träwels in one place! The rest is optional. 

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by MrKrisKrisu** [comment]
You can already view a map of trips for an entire day. -> https://traewelling.de/stats/daily/2023-06-17

A map with all journeys is technically easy to implement, but as you can already see on the [Active journeys page](https://traewelling.de/statuses/active), it is not very performant. Here we have to think about solutions first.

Routes have to be merged and polylines have to be reduced in their level of detail.

**Reply by dancojocaru2000** [reply]
Polylines aren't reduced in their level of detail on the Travelynx map as far as I can tell

**Comment by Fabian-C** [comment]
I recently build something like this.. it also has some performance problems but you may be able to retrieve the geojson data from the indexedDB storage to build your own map https://fabian-c.github.io/Traewelling-Maps/docs

**Reply by ErikUden** [reply]
really cool!!!

**Comment by MrKrisKrisu** [comment]
Will be available in the next release in the experimental / beta layout.

**Reply by ErikUden** [reply]
this is all i ever wanted <3
